From 86b962e7c0de43b20b6210d059e4855ce87078bc Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 13 Jul 2017 16:20:08 -0700
Subject: [PATCH] Adjust the order of headers to fix build for musl

Fixes
kexec/ifdown.c:33:16: error: storage size of 'ifc' isn't known

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 kexec/ifdown.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kexec/ifdown.c b/kexec/ifdown.c
index 2215798..e13f777 100644
--- a/kexec/ifdown.c
+++ b/kexec/ifdown.c
@@ -16,8 +16,8 @@ char *v_ifdown = "@(#)ifdown.c  1.11  02-Jun-1998  miquels@cistron.nl";
 #include <sys/time.h>
 #include <errno.h>
 
-#include <net/if.h>
 #include <netinet/in.h>
+#include <net/if.h>
 
 #define MAX_IFS	64
 
-- 
2.13.2

